src/user/user.rpc.ts

```diff
@@ -14,15 +16,16 @@ const packageDefinition = protoLoader.loadSync(
         oneofs: true,
     });
 const users_proto = grpc.loadPackageDefinition(packageDefinition).users;
-const client = new users_proto.Users('user-service:50051',
-                                     grpc.credentials.createInsecure());
+const client = new users_proto.Users(config.users.endpoint,
+                                        grpc.credentials.createInsecure());
 export class UsersRpc {
 
```

Comment:
Extra empty line, either comment the function and then put an empty line above the comment, or delete the empty line completely.